### ISSUE TITLE
SonarQube authentication is required to display badges

### ISSUE THREAD

**Issue by Kemyke** [QualInsight/qualinsight-plugins-sonarqube-badges#85]
Badges are linked in the status page of the project Azure Devops dashboard. However badges are only displayed if the user is logged in to SonarQube. 

Is there a way to use a token and add it as query parameter to do the authentication independently from the user who is watching the status page?

**Comment by Luukgr** [comment]
I like the plugin, but if authentication is needed, it makes the plugin unusable. Is it an idea, to set a path where the images are dropped? Then we can just run a web service?

**Comment by carlspring** [comment]
@Luukgr : You can complain to Sonarsource about this. They're trying really hard to kill off the ecosystem of OSS plugins.

**Comment by dirien** [comment]
Complaning at Sonarsource is not helping at all... they just don't care.

Anybod know any alternatives to sonarqube with similar functionality?

**Comment by josecolella** [comment]
I agree, currently all the badges in the project are broken. Has anyone found any solutions?
